src/typechecker/FStar.TypeChecker.NBETerm.fst

```diff
@@ -543,6 +543,7 @@ let e_norm_step =
                      mkFV (lid_as_fv PC.steps_unfoldattr S.delta_constant None)
                           [] [as_arg (embed (e_list e_string) cb l)]
         | SE.ZetaFull -> mkFV (lid_as_fv PC.steps_zeta_full S.delta_constant None) [] []
+        | SE.Unascribe -> mkFV (lid_as_fv PC.steps_unascribe S.delta_constant None) [] []
     in
     let un cb (t0:t) : option SE.norm_step =
```

Comment:
`ZetaFull` is not mentioned in this function, so I did not add `Unascribe`. Maybe they should both be added ?

Reply:
Yes, missing ZetaFull seems to be an oversight.